tests/unit/services/LibraryService.test.js

```diff
@@ -40,11 +40,19 @@ describe('Test: LibraryService', () => {
         version: '0.1.0',
       }];
 
-      const { api } = await import('boot/axios');
       api.get.mockImplementation(() => Promise.resolve({ data: { results: libraries } }));
 
       const data = await LibraryService.find();
       expect(data).toEqual(libraries);
     });
   });
+
+  describe('Test function: remove', () => {
+    it('should delete library', async () => {
```

Comment:
should remove the library corresponding to the given id

Reply:
done